src/services/helpers/gantt/ganttCell/GanttCell.tsx

```diff
@@ -35,13 +38,23 @@ export function GanttCell({
     color = "transparent";
     borderRadius = ["0", "0", "0", "0"];
     width = 100;
+
     rightAlign = true; // if true then align the cell right (the starting gantt cell in this case)
     if (isRoleAssignmentInTimeline(roleAssignment, timeline, index)) {
       if ("startConfidence" in roleAssignment) {
         roleAssignment = roleAssignment as Role;
-        color = getConfidenceColor(roleAssignment.startConfidence);
+        let confidenceType = "startConfidence";
+        if (i === 1) {
```

Comment:
what's special about i === 1? Can you add a comment?

Reply:
will add a comment.

This is due to the "duplicate role" implementation to render both start and end confidences correctly. (see getRolesAsRow fn) 

as noted in the PR, this should definitely be refactored in the future